examples/bri-3/src/bri/auth/guards/didJwt.guard.ts

```diff
@@ -23,6 +28,14 @@ export class DidJwtAuthGuard implements CanActivate {
     try {
       const token = this.getToken(request);
       const verified = await this.verifyJwt(token);
+      // TODO: we need to use did instead of public key in db
+      // a bit hacky way to attach bpiSubject to current request
+      const bpiSubject =
+        await this.bpiSubjectStorageAgent.getBpiSubjectByPublicKey(
```

Comment:
could we please add in comments, what is actually happening here? And why specifically 13 is being used?

Reply:
Also, better to eliminate any possible undefined values